### PR TITLE
Additional updates to support puppet-pulp 1.0.0

### PR DESCRIPTION
**Pull request by bbuckingham**
This commit contains some additional changes to support puppet-pulp 1.0.0
on a capsule.  This includes setting various pulp parameters to the
default values that katello/capsule had previously as well as inclusion
of the httpd conf files.

### CODE REVIEW

**Comment by bbuckingham** [comment]
Note: this PR also requires https://github.com/Katello/puppet-pulp/pull/109, which addresses a possible error with pulp-manage-db

File: manifests/config.pp [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Capsule Config
+class capsule::config {
+
+  file {'/etc/httpd/conf.d/pulp.conf':
```
Comment (ehelms) [inline manifests/config.pp]: We should wrap these delcarations in an `if $pulp` check

**Comment by bbuckingham** [comment]
@ehelms, updated

**Comment by ehelms** [comment]
ACK

**Pull request merged**